modules/oadp-restore-request-limit.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-restore-request-limit_{context}"]
+= Pods crash or restart due to lack of memory or CPU
+
+You can set specific resource limits and requests for memory or CPU if a Velero or Restic pod crashes due to a lack of either of them.
+
+== Velero pod
+
+Use the `configuration.velero.podConfig.resourceAllocations` specification field in the `oadp_v1alpha1_dpa.yaml` file to set specific resource limits and requests for a Velero pod.
+
+.Example Velero `yaml` file:
+
+[source,yaml]
+----
+  configuration:
+    velero:
+      podConfig:
+        resourceAllocations:
+          limits:
+            cpu: "2"
+            memory: 512Mi
+          requests:
+            cpu: 500m
+            memory: 256Mi
+----
+
+== Restic pod
+
+Use the `configuration.restic.podConfig.resourceAllocations` specification field to set  specific resource limits and requests for a Restic pod.
+
+.Example Restic `yaml` file:
+
+[source,yaml]
+----
+  configuration:
+    restic:
+      podConfig:
+        resourceAllocations:
+          limits:
+            cpu: "2"
+            memory: 512Mi
+          requests:
+            cpu: 500m
+            memory: 256Mi
+----
+
+[NOTE]
+====
+The values for the resource requests and limits flags follow the same format as Kubernetes resource requirements.
+
+Also, if you do not define `configuration.velero.podConfig.resourceAllocations` or `configuration.restic.podConfig.resourceAllocations`, then the default resources specification for a Velero or a Restic pod is:
+
+[source,yaml]
+----
+resources:
+  limits:
```

Comment:
While reviewing this PR, we realized we actually shouldn't be setting default limits and only requests.

This way a user won't hit arbitrary limits that they didn't set themselves.

Please remove the `limits` struct from this section as the only default will be `requests`

See https://issues.redhat.com/browse/OADP-648 for more info

Reply:
PR to remove limits set https://github.com/openshift/oadp-operator/pull/773